Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/Revolvers/marine_revolvers.yml

```diff
@@ -0,0 +1,101 @@
+- type: entity
+  parent: [BaseItem, CMBaseWeaponGun, RMCBaseAttachableHolder]
+  abstract: true
+  id: RMCWeaponRevolverBase
+  components:
+  - type: AmmoCounter
+  - type: Sprite
+    state: icon
+  - type: Item
+    size: Normal
+  - type: Tag
+    tags:
+    - Sidearm
+  - type: Clothing
+    slots:
+    - suitStorage
+    - belt
+  - type: Wieldable
+  - type: Gun
+    fireRate: 1.5
+    minAngle: 10
+    maxAngle: 20
+    angleIncrease: 5
+    angleDecay: 10
+    selectedMode: SemiAuto
+    availableModes:
+    - SemiAuto
+    soundGunshot:
+      path: /Audio/_RMC14/Weapons/Guns/Gunshots/gun_44mag.ogg
+  - type: GunWieldBonus
+    minAngle: -10
+    maxAngle: -10
+    angleIncrease: -4
+    angleDecay: 0
+  - type: ContainerContainer
+    containers:
+      revolver-ammo: !type:Container
+  - type: RevolverAmmoProvider
+    whitelist:
+      tags:
+        - RMCMagazineRevolver
+        - RMCSpeedLoaderM44
+        - RMCCartridgeRevolver44
+    proto: RMCCartridgeRevolver44
+    capacity: 6
+    chambers: [ True, True, True, True, True, True ]
+    ammoSlots: [ null, null, null, null, null, null ]
+    soundEject:
+      path: /Audio/Weapons/Guns/MagOut/revolver_magout.ogg
+    soundInsert:
+      path: /Audio/Weapons/Guns/MagIn/revolver_magin.ogg
+  - type: Appearance
+  - type: RMCNameItemOnVend
+    item: Sidearm
+  - type: GunUnskilledPenalty
+    angleIncrease: 5
+
+- type: entity
+  parent: RMCWeaponRevolverBase
+  id: RMCWeaponRevolverM44
+  name: M44 combat revolver
+  description: A bulky revolver, occasionally carried by assault troops and officers in the Marine Corps, as well as civilian law enforcement. Fires .44 Magnum rounds.
+  components:
+  - type: Gun
+    fireRate: 2.5
```

Comment:
The ROF is wrong. The M44's fire delay is `FIRE_DELAY_TIER_7`, which is 0.5 and equals a fire rate of 2.0.

Reply:
fixed